src/main/scala/com/goyeau/kubernetes/client/api/PodsApi.scala

```diff
@@ -32,7 +31,8 @@ private[client] case class PodsApi[F[_]](httpClient: Client[F], config: KubeConf
   val F: Async[F],
   val listDecoder: Decoder[PodList],
   encoder: Encoder[Pod],
-  decoder: Decoder[Pod]
+  decoder: Decoder[Pod],
+  contextShift: ContextShift[IO]
```

Comment:
```suggestion
  contextShift: ContextShift[F]
```
With this we are not forcing the IO monad

Reply:
No need anymore. Changed to `Async`